.github/workflows/e2e-android.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           yes | sdkmanager "platform-tools" "platforms;android-${{ inputs.android-api-level }}"
           set -o pipefail
 
+          # Install Ninja
```

Comment:
See [here](https://github.com/mobilestack-xyz/mobilestack-runtime/pull/1/files#r1756973682); not sure why this is now needed, though I have to imagine it's related to the rest of the changes in this PR